tests/test_cli.py

```diff
@@ -198,6 +198,43 @@ def test_mlflow_gc_sqlite(sqlite_store, create_artifacts_in_run):
     assert not os.path.exists(artifact_path)
 
 
+def test_mlflow_gc_sqlite_older_than(sqlite_store):
+    store = sqlite_store[0]
+    run = _create_run_in_store(store)
+    store.delete_run(run.info.run_uuid)
+    with pytest.raises(subprocess.CalledProcessError, match=r".+"):
```

Comment:
Can we include the expected error message in `match`?

```suggestion
    with pytest.raises(subprocess.CalledProcessError, match=r"Run \w+ is not older than the required age."):
```

Reply:
The code above doesn't seem to work, but the code below does:

```python
    with pytest.raises(subprocess.CalledProcessError, match=r".+") as exp:
        subprocess.run(
            [
                "mlflow",
                "gc",
                "--backend-store-uri",
                sqlite_store[1],
                "--older-than",
                "10d10h10m10s",
                "--run-ids",
                run.info.run_uuid,
            ],
            check=True,
            capture_output=True,
            text=True,
        )
    assert "is not older than the required age" in exp.value.stderr
```

Reply:
I have replaced the code with the snippet you have provided for both gc test case with file store and sqlite store